arq/connections.py

```diff
@@ -38,6 +39,21 @@ class RedisSettings:
     sentinel: bool = False
     sentinel_master: str = 'mymaster'
 
+    @classmethod
+    def from_redis_url(cls, url: str) -> 'RedisSettings':
+        """Parse a redis: connection URI, return a new
+        arq.connections.RedisSettings instance
+
+        :param url: The redis: url to parse
+        :return: :class:`arq.connections.RedisSettings` instance
+        :raises: AssertionError: if url is malformed
+            NOTE: Validation performed by `aioredis.util.parse_url` via
+            `assert` statements, and thus no validation takes place if various
+            Python interpreter optimizations are enabled at runtime.
+        """
+        address, options = parse_url(url)
+        return cls(host=address[0], port=address[1], password=options.get('password'))
```

Comment:
you should be able to set the database here too. see [here](https://github.com/samuelcolvin/foxglove/blob/22723a32b3a2c7ba8db24af8c7f2a3d0bb218b10/foxglove/settings.py#L162)

Reply:
I'm adding a kwargs passthru to `RedisSettings` constructor for remaining args (see subsequent commits)

Reply:
> you should be able to set the database here too

I trust I've addressed your concern here (please correct me if I'm wrong).  So I'm gong to resolve this conversation.